modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc

```diff
@@ -161,6 +163,39 @@ Your changes are saved automatically.
 
 NOTE: When you create a new mountpoint using Login credentials, you must log out of ownCloud and then log back in so you can access the share. You only have to do this the first time.
 
+==== Access Control Lists 
+
+From version 1.0.1 the Windows Network Drives App supports {acl-url}[Access Control Lists (ACLs)].
+To obtain ACL information, it supports two ACL providers:
+
+- xref:the-null-permission-manager[The Null Permission Manager]
+- xref:the-owncloud-ldap-permission-manager[The ownCloud LDAP Permission Manager] 
+
+image::enterprise/external_storage/windows_network_drive/acl-permissions-manager.png[Configuring ACL retrieval in the ownCloud Windows Network Drive app.]
+
+===== The Null Permission Manager
+
+This is the default permission manager, used if no other ACL manager is specified.
+If you want to retain ownCloud’s current behaviour, then use this permission manager. 
+When in effect, the Windows Network Drive app will use a file’s attributes (e.g., read-only, and hidden), to determine how the user can interact with the file.
+There are no usage restrictions.
+
+===== The ownCloud LDAP Permission Manager
+
+To use ACLs from an LDAP server, whether that is a Windows Active Directory server or a Samba server, use the ownCloud LDAP Permission Manager.
+To use this provider requires two, key, things:
+
+- An Active Directory server containing standard user and group information that can be used by {ldap-app-url}[ownCloud's LDAP app].
+- ownCloud's LDAP app to be properly configured to retrieve user and group information from the Active Directory server.
+
+IMPORTANT: The ownCloud LDAP apps must configure the `sAMAccountName` to be the ownCloud server's username.
+
```

Comment:
Adding some information:
The ownCloud LDAP Permission Manager will evaluate the ACL in the file along with the file attributes in order to determine the permissions. Both will be evaluated